test/run_unit_tests.sh

```diff
@@ -80,6 +80,8 @@ run_unittest test/unit/test_create_readnl_files.py
 run_unittest test/unit/test_write_init_files.py
 # ParamGen atm_in namelist writer unit tests:
 run_unittest test/unit/test_atm_in_paramgen.py
+# CAM history config unit tests
+run_unittest test/unit/test_hist_config.py
```

Comment:
I realized that we should probably add a `run_doctest cime_config/hist_config.py` entry to this shell script as well (see the entries above), as otherwise the doctests won't be run when a user or developer runs this script.

Reply:
good call - added!